permissions/src/iosMain/kotlin/dev/icerock/moko/permissions/ios/PermissionsController.kt

```diff
@@ -74,11 +77,14 @@ class PermissionsController : PermissionsControllerProtocol {
                     kCLAuthorizationStatusAuthorizedWhenInUse
                 ).contains(CLLocationManager.authorizationStatus())
             }
-            Permission.BLUETOOTH_LE -> true
             Permission.REMOTE_NOTIFICATION -> UIApplication.sharedApplication().registeredForRemoteNotifications
             Permission.RECORD_AUDIO -> AVCaptureDevice.authorizationStatusForMediaType(
                 AVMediaTypeAudio
             ) == AVAuthorizationStatusAuthorized
+            Permission.BLUETOOTH_LE -> true
+            Permission.BLUETOOTH_SCAN -> true
+            Permission.BLUETOOTH_ADVERTISE -> true
+            Permission.BLUETOOTH_CONNECT -> true
```

Comment:
https://developer.apple.com/documentation/corebluetooth/cbmanager/3377595-authorization

Reply:
Thank you for your feedback ! I've done it with compatibility on versions inferior than iOS 13 because I'll need it for some projects.